phpstan-baseline.neon

```diff
@@ -1300,6 +1300,11 @@ parameters:
 			count: 1
 			path: LibreNMS/Authentication/LdapAuthorizationAuthorizer.php
 
+		-
```

Comment:
Sorry we only support removing from this file, please type hint the parameter

Reply:
Thanks for the feedback. I've added another commit to incorporate your suggestion.